source/cloud/cloud-administration/site-configuration.rst

```diff
@@ -32,27 +32,27 @@ Mattermost in-product notices keep users and administrators informed of the newe
 Administrator Notices
 ^^^^^^^^^^^^^^^^^^^^^
 
-Administrator notices are used to inform System Admins when a new server version is available, or if a server upgrade is recommended due to ending support life cycles. System Admins may also receive notices about recommended server configuration options to optimize the user experience of their deployment.
+Administrator notices are used to inform System Admins of important updates or recommended configuration options to optimize the user experience of their deployment.
 
-Administrator notices can be disabled in the **System Console > Notices** page.
+Administrator notices can be disabled in the **System Console > Site Configuration > Notices** page.
 
 End User Notices
 ^^^^^^^^^^^^^^^^
 
-End user notices are used to inform users and Admins of new feature enhancements and when new desktop versions are available. They can be disabled in **System Console > Notices**.
+End user notices are used to inform users and Admins of new feature enhancements and when new desktop versions are available. They can be disabled in **System Console > Site Configuration > Notices**.
 
 FAQs
 ^^^^
 
 Are notices enabled by default?
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-Notices are enabled by default for all Mattermost users. System Admins may choose to disable administrator or end user notices in **System Console > Notices**.
+Notices are enabled by default for all Mattermost users. System Admins may choose to disable administrator or end user notices in **System Console > Site Configuration > Notices**.
 
-Will I still receive notices if my server is air-gapped?
+Will I still receive notices if my workstation is air-gapped?
```

Comment:
Is this workstation or workspace? Both make sense in different contexts.

Reply:
@justinegeffen - it's "workspace" and not "workstation". Great catch! Thanks!

Reply:
Cloud workspaces by definition can't be air-gapped. Air-gapped means the system is physically isolated from the internet, and data can only pass to it via removable media such as a USB flash. We can remove this line.